### PR TITLE
fix: Solved issue of rising and falling edges in Logic Analyzer

### LINKED ISSUE #1153: Rising and Falling edges aren't plotted properly
**Actual Behaviour**

The rising and falling edges options plots all the edges irrespective of selection

**Expected Behaviour**

The output should match with the option selected

**Steps to reproduce it**

Go to Logic Analyzer -> Select `Two Channel Mode` -> keep ID1 for both channels and `RISING EDGES` for one and `FALLING EDGES` for other -> Analyze

**LogCat for the issue**

Not applicable

**Screenshots of the issue**

screenshot_20180627-140657__01

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by harsh-2711**
Fixes #1153 

**Changes**: Added different methods to handle the rising and falling edges in Logic Analyzer

**Screenshot/s for the changes**: `WIP`

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2177958/app-debug.zip)

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java [hunk 1]
```diff
@@ -524,6 +524,31 @@ private void singleChannelOtherEdges(double[] xData, double[] yData) {
         setLineDataSet();
     }
 
+    private void singleChannelRisingEdges(double[] xData, double[] yData) {
+
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java]: Let's remove all these empty lines. It's making the method body long.

**Comment by Vikum94** [comment]
@harsh-2711 Please add screenshots and steps on how to test this PR

**Comment by harsh-2711** [comment]
@Vikum94 I have requested @abhinavraj23 for these screenshots too. Will update by tomorrow

**Comment by harsh-2711** [comment]
@CloudyPadmal Changes done.

**Comment by harsh-2711** [comment]
@Vikum94 Steps are same as I have described in PR #1159 but the selection of mode can be any in this case. Also, use `RISING EDGES` or `FALLING EDGES` option to see the changes made in this PR.

**Pull request merged**